packages/core/admin/admin/src/hooks/useFetchMarketplaceProviders/utils/api.js

```diff
@@ -1,10 +1,11 @@
-import axios from 'axios';
+import { getFetchClient } from '@strapi/helper-plugin';
 import qs from 'qs';
 
 const MARKETPLACE_API_URL = 'https://market-api.strapi.io';
+const { get } = getFetchClient();
 
 const fetchMarketplaceProviders = async (params = {}) => {
```

Comment:
What do you think about removing this whole utility and inline it into the hook here? https://github.com/strapi/strapi/blob/ac5610251e1bb9d602fdb78c3e98bdad3f65eb42/packages/core/admin/admin/src/hooks/useFetchMarketplaceProviders/index.js This would be much more in line with the rest of our code.

Reply:
yes sure

Reply:
done